php/README.md

```diff
@@ -0,0 +1,141 @@
+# PHP
+
+## PSR-2
+
+To increase the readability of all the code, it is a good idea to go towards a standard that we all conform to. The PHP world has developed a nice standard for this: The PSR-2 standard. For the most used IDEs I will post a link here for how to set this up:
```

Comment:
Please add a link to the PSR-2 standards

Reply:
Fixed.

Reply:
I still don't see a link